### PR TITLE
Chore/move cmdk

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

- move cmdk to ui-patterns. 
- ready to change over primitives to use `<Command>` from `ui` package

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Review by joshenlim — APPROVED** [review]

File: packages/ui/tsconfig.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "resolveJsonModule": true
   },
-  "include": ["."],
```
Comment (kevcodez) [inline packages/ui/tsconfig.json:6]: Do we need this?

File: packages/ui-patterns/package.json [hunk 2]
```diff
@@ -16,7 +16,10 @@
     "react-tooltip": "*",
     "reactflow": "*",
     "tsconfig": "*",
-    "ui": "*"
+    "ui": "*",
+    "cmdk": "^0.2.1",
+    "react-error-boundary": "^4.0.12",
+    "@supabase/auth-helpers-react": "*"
```
Comment (kevcodez) [inline packages/ui-patterns/package.json:22]: We previously had this at `^0.4.2` - do we wanna use a wildcard here now?

**Review by kevcodez — APPROVED** [review]

File: packages/ui/tsconfig.json (hunk 1 above)
Comment (MildTomato) [reply packages/ui/tsconfig.json:6]: oops reverting

File: packages/ui-patterns/package.json (hunk 2 above)
Comment (MildTomato) [reply packages/ui-patterns/package.json:22]: duno. i thought we use wildcards in here since they are installed in the apps

Comment (MildTomato) [reply packages/ui-patterns/package.json]: ill use the numbered version

**Review by fsansalvadore — APPROVED** [review]

**Pull request merged**